contributing_to_docs/term_glossary.adoc

```diff
@@ -423,16 +423,13 @@ When referencing as a prerequisite for a procedure module, use the following
 construction: Install the OpenShift CLI (`oc`).
 
 ''''
-=== OpenShift control plane (also known as master)
+=== OpenShift control plane
```

Comment:
We should just refer to this as "control plane" and drop "OpenShift" for better alphabetization. Otherwise, LGTM. Thanks!